app/controllers/ems_common.rb

```diff
@@ -109,8 +109,8 @@ def view_setup_params
     }
   end
 
-  def show_entities(display)
-    view_setup_helper(display, *view_setup_params[display])
+  def show_entities(disp)
```

Comment:
@enoodle why do we have to change parameter name, looks like this change is not required